team/team.go

```diff
@@ -32,6 +34,37 @@ func LoadTeams(fluidkeysDirectory string) ([]Team, error) {
 	return teams, nil
 }
 
+// Save validates the given team then tries to create a toml team roster in a subdirectory of the
+// given directory.
+func Save(team Team, fluidkeysDirectory string) error {
+	err := team.Validate()
+	if err != nil {
+		return fmt.Errorf("invalid team: %v", err)
+	}
+	rosterDirectory := filepath.Join(
+		getTeamDirectory(fluidkeysDirectory), // ~/.config/fluidkeys/teams
+		team.subDirectory(),                  // fluidkeys-inc-4367436743
+	)
+	if _, err := os.Stat(rosterDirectory); !os.IsNotExist(err) {
+		return fmt.Errorf("path already exists at %s", rosterDirectory)
```

Comment:
I'm not sure why you'd want to raise an error if the subdirectory already exists? That would mean `Save` can only work exactly once, right?